UNRELEASED.md

```diff
@@ -12,4 +12,5 @@
      DO NOT LEAVE A BLANK LINE BELOW THIS PREAMBLE -->
 ### Bug fixes
 
-* Fix function application static check in arrays encoding, see #1490
\ No newline at end of file
+* Fix function application static check in arrays encoding, see #1490
```

Comment:
What's the change in this line? Is it some weird white-space thing?

Reply:
perhaps formatting? No idea

Reply:
Aaah, it was visible in the emailed diff; line before didn't end in a newline!